odoo/addons/base/ir/ir_qweb/assetsbundle.py

```diff
@@ -480,7 +480,7 @@ def _fetch_content(self):
                 with open(self._filename, 'rb') as fp:
                     return fp.read().decode('utf-8')
             else:
-                return self._ir_attach['datas'].decode('base64')
+                return self._ir_attach['datas'].decode('base64').decode('utf-8')
```

Comment:
@Gorash What do you think about this ? I think this code is only used to serve .less.css generated files but for some reason, with the LESS editor some files does not work if I do not encode them as utf-8 (and I think that following the `fp.read().decode('utf-8')` above, it seems consistent).